templates/workspace_services/guacamole/guacamole-server/guacamole-auth-azure/pom.xml

```diff
@@ -138,6 +138,11 @@
                     </execution>
                 </executions>
             </plugin>
+           <plugin>
+             <groupId>org.apache.maven.plugins</groupId>
+             <artifactId>maven-surefire-plugin</artifactId>
+             <version>2.22.1</version>
```

Comment:
This version is 3 years old and there is a more current one. I'll let Elad decide if he can get into this.

Reply:
I changed it and tested it on the newest version now.